stdlib/public/stubs/LibcShims.cpp

```diff
@@ -155,7 +155,7 @@ double swift::_swift_stdlib_remainder(double dividend, double divisor) {
 SWIFT_RUNTIME_STDLIB_INTERFACE
 double swift::_swift_stdlib_squareRoot(double x) { return std::sqrt(x); }
 
-#if (defined __i386__ || defined __x86_64__) && !defined _MSC_VER
+#if (defined __i386__ || defined __x86_64__) && !defined _WIN32
```

Comment:
Can you change this to:

    #if (defined(__i386__) || defined(__x86_64__)) && !defined(_WIN32)

to match everything else?